client/ayon_houdini/plugins/publish/collect_asset_handles.py

```diff
@@ -50,9 +50,9 @@ def process(self, instance):
 
         attr_values = self.get_attr_values_from_data(instance.data)
         if attr_values.get("use_handles", self.use_asset_handles):
-            folder_attributes = instance.data["folderEntity"]["attrib"]
-            handle_start = folder_attributes.get("handleStart", 0)
-            handle_end = folder_attributes.get("handleEnd", 0)
+            task_attributes = instance.data["taskEntity"]["attrib"]
```

Comment:
This may need to have a fallback to `folderEntity` because `task` may not be required for publish instances. Correct @iLLiCiTiT ?

Does that mean the 'label' changes here from Folder to Task across these plug-ins is actually counter intuitive? Any better names we can think of that represents' the Instance's folder/task attributes?

Reply:
There is `ValidateInstanceInContextHoudini` but it is optional, so probably yes.